wormhole-connect/src/hooks/useTrackTransfer.ts

```diff
@@ -18,37 +18,46 @@ const useTrackTransfer = (): void => {
 
     const track = async () => {
       const { route, receipt } = routeContext;
+
       if (!route || !receipt) {
         return;
       }
+
       let stateChanged = false;
+
       while (isActive && !isCompleted(receipt) && !stateChanged) {
+        console.log(`Current state: ${receipt.state}`);
```

Comment:
Do we want to keep these console logs?

Reply:
I moved them outside of try/catch, but I don't think we need to keep them. @artursapek ?

Reply:
Removed 👍